src/toolkit/position.cycpp.h

```diff
@@ -28,5 +25,14 @@ std::vector<int> cycpp_shape_latitude = 0;
        "be expressed in degrees as a double." \
 }
 double longitude;
+
+protected:
```

Comment:
Shall we also enforce the visibility of the `coordinates`, `latitude ` and `longitude` variable ?

i.e. if `InitializePosition` is `protected`, shall not `coordinates` to also be `private` ?
What about `lat` and `long` ?

Reply:
In my opinion she shall define the injection as "protected/private/public" in the archetype, as:
```
private:
   // Code Injection:
   #include "toolkit/position.cycpp.h"
```
this would allow the developer of the archetype to control this and not be enforced and potentially have impact on the rest of the archetype members